Marlin/configuration_store.cpp

```diff
@@ -344,10 +349,12 @@ void Config_RetrieveSettings() {
 
     uint8_t dummy_uint8 = 0, mesh_num_x = 0, mesh_num_y = 0;
     EEPROM_READ_VAR(i, dummy_uint8);
+    EEPROM_READ_VAR(i, dummy);
```

Comment:
Same here. See how we've preset the written dummy values to zero. Here, `dummy` is some "unknown" float. (Side note - Perhaps we should rename it `dummy_float` so we remember it's a float.)